06 mars 2017 16:06:56,691 INFO  ~ Starting /home/formation/workspace/projet_betclic/betclicV2/.
06 mars 2017 16:06:57,953 WARN  ~ You're running Play! in DEV mode
06 mars 2017 16:07:00,550 INFO  ~ Connected to jdbc:h2:mem:play;MODE=MYSQL;LOCK_MODE=0 for default
06 mars 2017 16:07:02,164 DEBUG ~ 
    drop table Bet if exists
06 mars 2017 16:07:02,165 DEBUG ~ 
    drop table BetResult if exists
06 mars 2017 16:07:02,166 DEBUG ~ 
    drop table BetType if exists
06 mars 2017 16:07:02,166 DEBUG ~ 
    drop table Cote if exists
06 mars 2017 16:07:02,166 DEBUG ~ 
    drop table DoBet if exists
06 mars 2017 16:07:02,166 DEBUG ~ 
    drop table Meeting if exists
06 mars 2017 16:07:02,167 DEBUG ~ 
    drop table User if exists
06 mars 2017 16:07:02,167 DEBUG ~ 
    drop table UserContact if exists
06 mars 2017 16:07:02,167 DEBUG ~ 
    create table Bet (
        id bigint generated by default as identity,
        name varchar(255) not null,
        status varchar(255),
        betTypeId bigint,
        coteWin_id bigint,
        meetingId bigint,
        primary key (id)
    )
06 mars 2017 16:07:02,181 DEBUG ~ 
    create table BetResult (
        id bigint generated by default as identity,
        montant decimal(19,2) not null,
        status integer not null,
        primary key (id)
    )
06 mars 2017 16:07:02,185 DEBUG ~ 
    create table BetType (
        id bigint generated by default as identity,
        name varchar(255) not null,
        primary key (id)
    )
06 mars 2017 16:07:02,187 DEBUG ~ 
    create table Cote (
        id bigint generated by default as identity,
        cote double not null,
        name varchar(255) not null,
        betId bigint,
        primary key (id)
    )
06 mars 2017 16:07:02,190 DEBUG ~ 
    create table DoBet (
        id bigint generated by default as identity,
        montant decimal(19,2) not null,
        cote_id bigint,
        result_id bigint,
        user_id bigint,
        betId bigint,
        primary key (id)
    )
06 mars 2017 16:07:02,195 DEBUG ~ 
    create table Meeting (
        id bigint generated by default as identity,
        date timestamp not null,
        name varchar(255) not null,
        status varchar(255) not null,
        primary key (id)
    )
06 mars 2017 16:07:02,198 DEBUG ~ 
    create table User (
        id bigint generated by default as identity,
        email varchar(255) not null,
        firstname varchar(255) not null,
        lastname varchar(255) not null,
        password varchar(255) not null,
        primary key (id)
    )
06 mars 2017 16:07:02,202 DEBUG ~ 
    create table UserContact (
        id bigint generated by default as identity,
        email varchar(255) not null,
        firstname varchar(255) not null,
        lastname varchar(255) not null,
        message varchar(255) not null,
        primary key (id)
    )
06 mars 2017 16:07:02,205 DEBUG ~ 
    alter table User 
        add constraint UK_e6gkqunxajvyxl5uctpl2vl2p unique (email)
06 mars 2017 16:07:02,212 DEBUG ~ 
    alter table Bet 
        add constraint FK_f5pvc5isely27h7vd2sqkqe6v 
        foreign key (betTypeId) 
        references BetType
06 mars 2017 16:07:02,228 DEBUG ~ 
    alter table Bet 
        add constraint FK_dpn9go2iinf0rsi0ko3j18sie 
        foreign key (coteWin_id) 
        references Cote
06 mars 2017 16:07:02,231 DEBUG ~ 
    alter table Bet 
        add constraint FK_nn0425yobxqdqxjban3yb1tom 
        foreign key (meetingId) 
        references Meeting
06 mars 2017 16:07:02,235 DEBUG ~ 
    alter table Cote 
        add constraint FK_293uau4e3i1kd7psawne1sl1o 
        foreign key (betId) 
        references Bet
06 mars 2017 16:07:02,238 DEBUG ~ 
    alter table DoBet 
        add constraint FK_i7jsu1rxxbretfu3jbqaapr62 
        foreign key (cote_id) 
        references Cote
06 mars 2017 16:07:02,241 DEBUG ~ 
    alter table DoBet 
        add constraint FK_6f0krja6rt1kmqw2j9ppm5xdv 
        foreign key (result_id) 
        references BetResult
06 mars 2017 16:07:02,243 DEBUG ~ 
    alter table DoBet 
        add constraint FK_ppm39h9lga3tj23avp2x4uev6 
        foreign key (user_id) 
        references User
06 mars 2017 16:07:02,246 DEBUG ~ 
    alter table DoBet 
        add constraint FK_l4nnq5oxarnsbd3c2ckasoodk 
        foreign key (betId) 
        references Bet
06 mars 2017 16:07:02,333 INFO  ~ Application 'betclicV2' is now started !
06 mars 2017 16:07:04,720 DEBUG ~ 
    insert 
    into
        User
        (id, email, firstname, lastname, password) 
    values
        (null, ?, ?, ?, ?)
06 mars 2017 16:07:04,724 WARN  ~ SQL Error: 90121, SQLState: 90121
06 mars 2017 16:07:04,724 ERROR ~ Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185]
06 mars 2017 16:07:04,728 WARN  ~ [c3p0] A PooledConnection that has already signalled a Connection error is still in use!
06 mars 2017 16:07:04,729 WARN  ~ [c3p0] Another error has occurred [ org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185] ] which will not be reported to listeners!
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1454)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:487)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.rollback(NewProxyConnection.java:1033)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.doRollback(JdbcTransaction.java:163)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.rollback(AbstractTransactionImpl.java:209)
	at org.hibernate.ejb.TransactionImpl.rollback(TransactionImpl.java:106)
	at play.db.jpa.JPA.withTransaction(JPA.java:299)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:298)
	at play.jobs.Job.withinFilter(Job.java:185)
	at play.jobs.Job.call(Job.java:204)
	at play.jobs.Job.run(Job.java:172)
	at play.jobs.JobsPlugin.afterApplicationStart(JobsPlugin.java:119)
	at play.plugins.PluginCollection.afterApplicationStart(PluginCollection.java:622)
	at play.Play.start(Play.java:559)
	at play.test.PlayJUnitRunner.<init>(PlayJUnitRunner.java:38)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.junit.internal.builders.AnnotatedBuilder.buildRunner(AnnotatedBuilder.java:104)
	at org.junit.internal.builders.AnnotatedBuilder.runnerForClass(AnnotatedBuilder.java:86)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:33)
	at org.junit.internal.requests.FilterRequest.getRunner(FilterRequest.java:36)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:49)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:51)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:237)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
06 mars 2017 16:07:04,732 WARN  ~ [c3p0] A PooledConnection that has already signalled a Connection error is still in use!
06 mars 2017 16:07:04,732 WARN  ~ [c3p0] Another error has occurred [ org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185] ] which will not be reported to listeners!
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1444)
	at org.h2.jdbc.JdbcConnection.setAutoCommit(JdbcConnection.java:428)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.setAutoCommit(NewProxyConnection.java:1085)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.releaseManagedConnection(JdbcTransaction.java:127)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.doRollback(JdbcTransaction.java:170)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.rollback(AbstractTransactionImpl.java:209)
	at org.hibernate.ejb.TransactionImpl.rollback(TransactionImpl.java:106)
	at play.db.jpa.JPA.withTransaction(JPA.java:299)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:298)
	at play.jobs.Job.withinFilter(Job.java:185)
	at play.jobs.Job.call(Job.java:204)
	at play.jobs.Job.run(Job.java:172)
	at play.jobs.JobsPlugin.afterApplicationStart(JobsPlugin.java:119)
	at play.plugins.PluginCollection.afterApplicationStart(PluginCollection.java:622)
	at play.Play.start(Play.java:559)
	at play.test.PlayJUnitRunner.<init>(PlayJUnitRunner.java:38)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.junit.internal.builders.AnnotatedBuilder.buildRunner(AnnotatedBuilder.java:104)
	at org.junit.internal.builders.AnnotatedBuilder.runnerForClass(AnnotatedBuilder.java:86)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:33)
	at org.junit.internal.requests.FilterRequest.getRunner(FilterRequest.java:36)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:49)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:51)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:237)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
06 mars 2017 16:07:04,737 WARN  ~ [c3p0] A PooledConnection that has already signalled a Connection error is still in use!
06 mars 2017 16:07:04,737 WARN  ~ [c3p0] Another error has occurred [ org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185] ] which will not be reported to listeners!
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1444)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:620)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.getWarnings(NewProxyConnection.java:935)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.handleAndClearWarnings(SqlExceptionHelper.java:264)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logAndClearWarnings(SqlExceptionHelper.java:246)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.releaseConnection(LogicalConnectionImpl.java:238)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.close(LogicalConnectionImpl.java:168)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.close(JdbcCoordinatorImpl.java:186)
	at org.hibernate.engine.transaction.internal.TransactionCoordinatorImpl.close(TransactionCoordinatorImpl.java:292)
	at org.hibernate.internal.SessionImpl.close(SessionImpl.java:354)
	at org.hibernate.ejb.EntityManagerImpl.close(EntityManagerImpl.java:137)
	at play.db.jpa.JPA.withTransaction(JPA.java:311)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:298)
	at play.jobs.Job.withinFilter(Job.java:185)
	at play.jobs.Job.call(Job.java:204)
	at play.jobs.Job.run(Job.java:172)
	at play.jobs.JobsPlugin.afterApplicationStart(JobsPlugin.java:119)
	at play.plugins.PluginCollection.afterApplicationStart(PluginCollection.java:622)
	at play.Play.start(Play.java:559)
	at play.test.PlayJUnitRunner.<init>(PlayJUnitRunner.java:38)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.junit.internal.builders.AnnotatedBuilder.buildRunner(AnnotatedBuilder.java:104)
	at org.junit.internal.builders.AnnotatedBuilder.runnerForClass(AnnotatedBuilder.java:86)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:33)
	at org.junit.internal.requests.FilterRequest.getRunner(FilterRequest.java:36)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:49)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:51)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:237)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
06 mars 2017 16:07:04,740 WARN  ~ [c3p0] A PooledConnection that has already signalled a Connection error is still in use!
06 mars 2017 16:07:04,741 WARN  ~ [c3p0] Another error has occurred [ org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185] ] which will not be reported to listeners!
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1444)
	at org.h2.jdbc.JdbcConnection.clearWarnings(JdbcConnection.java:634)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.clearWarnings(NewProxyConnection.java:959)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.handleAndClearWarnings(SqlExceptionHelper.java:271)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logAndClearWarnings(SqlExceptionHelper.java:246)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.releaseConnection(LogicalConnectionImpl.java:238)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.close(LogicalConnectionImpl.java:168)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.close(JdbcCoordinatorImpl.java:186)
	at org.hibernate.engine.transaction.internal.TransactionCoordinatorImpl.close(TransactionCoordinatorImpl.java:292)
	at org.hibernate.internal.SessionImpl.close(SessionImpl.java:354)
	at org.hibernate.ejb.EntityManagerImpl.close(EntityManagerImpl.java:137)
	at play.db.jpa.JPA.withTransaction(JPA.java:311)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:298)
	at play.jobs.Job.withinFilter(Job.java:185)
	at play.jobs.Job.call(Job.java:204)
	at play.jobs.Job.run(Job.java:172)
	at play.jobs.JobsPlugin.afterApplicationStart(JobsPlugin.java:119)
	at play.plugins.PluginCollection.afterApplicationStart(PluginCollection.java:622)
	at play.Play.start(Play.java:559)
	at play.test.PlayJUnitRunner.<init>(PlayJUnitRunner.java:38)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.junit.internal.builders.AnnotatedBuilder.buildRunner(AnnotatedBuilder.java:104)
	at org.junit.internal.builders.AnnotatedBuilder.runnerForClass(AnnotatedBuilder.java:86)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:33)
	at org.junit.internal.requests.FilterRequest.getRunner(FilterRequest.java:36)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:49)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:51)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:237)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
06 mars 2017 16:07:04,752 ERROR ~ 

@739cb5db4
Error during job execution (job.InitialDbImportJob)

Execution exception (In /app/job/InitialDbImportJob.java around line 15)
RuntimeException occured : Cannot load fixture initial-data.yml: org.hibernate.exception.GenericJDBCException: could not prepare statement

play.exceptions.JavaExecutionException: Cannot load fixture initial-data.yml: org.hibernate.exception.GenericJDBCException: could not prepare statement
	at play.jobs.Job.call(Job.java:225)
	at Invocation.Job(Play!)
Caused by: java.lang.RuntimeException: Cannot load fixture initial-data.yml: org.hibernate.exception.GenericJDBCException: could not prepare statement
	at play.test.Fixtures.loadModels(Fixtures.java:273)
	at play.test.Fixtures.loadModels(Fixtures.java:173)
	at job.InitialDbImportJob.doJob(InitialDbImportJob.java:15)
	at play.jobs.Job.doJobWithResult(Job.java:54)
	at play.jobs.Job$2.apply(Job.java:208)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:298)
	at play.jobs.Job.withinFilter(Job.java:185)
	at play.jobs.Job.call(Job.java:204)
	... 1 more
Caused by: javax.persistence.PersistenceException: org.hibernate.exception.GenericJDBCException: could not prepare statement
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1387)
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1310)
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1316)
	at org.hibernate.ejb.AbstractEntityManagerImpl.persist(AbstractEntityManagerImpl.java:881)
	at play.db.jpa.JPABase._save(JPABase.java:35)
	at play.test.Fixtures.loadModels(Fixtures.java:252)
	... 10 more
Caused by: org.hibernate.exception.GenericJDBCException: could not prepare statement
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:54)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:124)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:193)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareStatement(StatementPreparerImpl.java:115)
	at org.hibernate.id.IdentityGenerator$GetGeneratedKeysDelegate.prepare(IdentityGenerator.java:89)
	at org.hibernate.id.insert.AbstractReturningDelegate.performInsert(AbstractReturningDelegate.java:55)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:2987)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3499)
	at org.hibernate.action.internal.EntityIdentityInsertAction.execute(EntityIdentityInsertAction.java:81)
	at org.hibernate.engine.spi.ActionQueue.execute(ActionQueue.java:395)
	at org.hibernate.engine.spi.ActionQueue.addResolvedEntityInsertAction(ActionQueue.java:229)
	at org.hibernate.engine.spi.ActionQueue.addInsertAction(ActionQueue.java:209)
	at org.hibernate.engine.spi.ActionQueue.addAction(ActionQueue.java:193)
	at org.hibernate.event.internal.AbstractSaveEventListener.addInsertAction(AbstractSaveEventListener.java:321)
	at org.hibernate.event.internal.AbstractSaveEventListener.performSaveOrReplicate(AbstractSaveEventListener.java:286)
	at org.hibernate.event.internal.AbstractSaveEventListener.performSave(AbstractSaveEventListener.java:192)
	at org.hibernate.event.internal.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:125)
	at org.hibernate.ejb.event.EJB3PersistEventListener.saveWithGeneratedId(EJB3PersistEventListener.java:78)
	at org.hibernate.event.internal.DefaultPersistEventListener.entityIsTransient(DefaultPersistEventListener.java:208)
	at org.hibernate.event.internal.DefaultPersistEventListener.onPersist(DefaultPersistEventListener.java:151)
	at org.hibernate.event.internal.DefaultPersistEventListener.onPersist(DefaultPersistEventListener.java:78)
	at org.hibernate.internal.SessionImpl.firePersist(SessionImpl.java:772)
	at org.hibernate.internal.SessionImpl.persist(SessionImpl.java:746)
	at org.hibernate.internal.SessionImpl.persist(SessionImpl.java:750)
	at org.hibernate.ejb.AbstractEntityManagerImpl.persist(AbstractEntityManagerImpl.java:875)
	... 12 more
Caused by: org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1444)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:275)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:1123)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$2.doPrepare(StatementPreparerImpl.java:117)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:183)
	... 34 more
